client/public/src/models/PersonModel.js

```diff
@@ -213,7 +213,7 @@ class PersonModel extends BaseModel {
       }
     }
 
-    if( res.length === 0 ) {
+    if( type !== 'oap' && res.length === 0 ) {
```

Comment:
I guess this should be `type === odr` cause that's the only time we'd want to do it.  It's dumb logic, because it's what we want to do by default, but default sets the type to odr

Reply:
Ok thanks, I've updated the code.